/**:
  ros__parameters:
    frame_update_hz: 0.033            # Frequency to perform alignment

    voxel_resolution: 1.0

    verbose: true
    loop:
      verbose: true

    local_reg:
      num_threads: 8                  # Number of threads used for local registration
      correspondences_number: 20      # Minimum number of correspondences required
      max_num_iter: 32                # Maximum iterations for local registration
      scale_factor_for_corr_dist: 3.0 # ⭐ Scale for correspondence search distance
      overlap_threshold: 0.0          # Not used in the alignment. `num_inliers_threshold` is enough.

    global_reg:
      enable: true
      # NOTE(hlim): The best value of `num_inliers_threshold` highly depends on `num_submap_keyframes`
      # Thus, heuristic tuning is required for the best performance.
      # Larger values make it more conservative (i.e., higher `num_inliers_threshold` enhances LC precision)
      num_inliers_threshold: 20       # ⭐ Minimum inlier count to accept KISS-Matcher initial alignment
